app/reducers/index.js

```diff
@@ -167,9 +169,11 @@ export const galleryPictureSchema = new schema.Entity('galleryPictures', {
 export const gallerySchema = new schema.Entity('galleries');
 
 export const quoteSchema = new schema.Entity('quotes', {
-  comments: [commentSchema]
+  reactions: [reactionSchema]
 });
 
+export const randomQuoteSchema = new schema.Entity('randomQuote');
```

Comment:
Adding this as a separate schema entity seems a little overkill. Maybe just add `randomQuoteSchema` as a key in the Quote entity reducer instead? 😄

Reply:
Agreed, done 👍